default_speed_x: 0.2
default_speed_y: 0.2
default_turn_speed: 0.5
